tools/api-markdown-documenter/prettier.config.cjs

```diff
@@ -5,6 +5,7 @@
 
 module.exports = {
     ...require("@fluidframework/build-common/prettier.config.cjs"),
+    useTabs: false, // @fluidframework/build-common ^1.2.0, change to TRUE after prettier infra
```

Comment:
this seems unrelated.

Reply:
also comment looks truncated?

Reply:
I made a note about this in the PR description, but extracting this into its own PR is probably better. I'll do that.

Reply:
🤦‍♂️This was only required because I was in symlinked mode... Changes reverted.